### ISSUE TITLE
Crash by data detector when setting text to nil

### ISSUE THREAD

**Issue by galambalazs** [TTTAttributedLabel/TTTAttributedLabel#546]
````*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSDataDetector enumerateMatchesInString:options:range:usingBlock:]: nil argument'
*** First throw call stack:
(0x1840682d8 0x19583c0e4 0x184068218 0x184feda58 0x184ef7634 0x1000679f0 0x100120fd4 0x100120f94 0x10012eb54 0x100130248 0x19606d22c 0x19606cef0)
libc++abi.dylib: terminating with uncaught exception of type NSException````

Offending line: https://github.com/TTTAttributedLabel/TTTAttributedLabel/blob/master/TTTAttributedLabel/TTTAttributedLabel.m#L1135

see also: https://github.com/jverkoey/nimbus/issues/160

**Comment by nikhilolx** [comment]
@mattt  – When will this commit be available on pod? The latest pod version 1.13.4 does not contain this fix and I've found this to be a hugely recurring crash (through Crashlytics).

**Comment by galambalazs** [comment]
+1

@nikhilolx in the meantime you can use this syntax to use a specific commit:

pod 'Y', :git => 'https://github.com/NSHipster/Y.git', :commit => 'b4dc0ffee'

http://nshipster.com/cocoapods/
